docs/configuration.md

```diff
@@ -0,0 +1,198 @@
+# Lighthouse Configuration
+
+The Lighthouse config object is the primary method of customizing a Lighthouse run to suit your use case. Using a custom config, you can limit the audits to run, add additional loads of the page under special conditions, add your own custom checks, tweak the scoring, and more.
+
+## Usage
+
+You can specify a custom config file when using lighthouse through the CLi or consuming the npm module yourself.
+
+**CLI**
+```sh
+lighthouse --config-path=path/to/custom-config.js https://example.com
+```
+
+**Node**
+```js
+const lighthouse = require('lighthouse')
+
+const config = {
+  passes: [{
+    recordTrace: true,
+    pauseBeforeTraceEndMs: 5000,
+    useThrottling: true,
+    gatherers: [],
+  }],
+
+  audits: [
+    'first-meaningful-paint',
+    'speed-index-metric',
+    'estimated-input-latency',
+    'first-interactive',
+    'consistently-interactive',
+  ]
+}
+
+lighthouse('http://example.com/', {port: 9222}, config)
+```
+
+## Properties
+
+### `extends: string|boolean|undefined`
+
+The extends property controls if your configuration should inherit from the default Lighthouse configuration. [Learn more.](#config-extension)
+
+#### Example
+```js
+{
+  extends: 'lighthouse:default',
+}
+```
+
+### `settings: Object|undefined`
+
+The settings property controls various aspects of running Lighthouse such as CPU/network throttling and audit whitelisting/blacklisting.
+
+#### Example
+```js
+{
+  settings: {
+    onlyCategories: ['performance'],
+    onlyAudits: ['works-offline'],
+  }
+}
+```
+
+#### Schema
+| Name | Type | Description |
+| -- | -- | -- |
+| onlyCategories | `string[]` | Limits the run to just what's required for the specified categories. Additive with `onlyAudits`. |
+| onlyAudits | `string[]` | Limits the run to just what's required for the specified audits. Additive with `onlyCategories`. |
+| skipAudits | `string[]` | Prevents the specified audits from being evaluated in the run. Takes priority over `onlyCategories`, not usable in conjuction with `onlyAudits`. |
+
+### `passes: Object[]`
+
+The passes property controls how to load the requested URL and what information to gather about the page while loading. Each entry in the passes array represents one load of the page (e.g. 4 entries in `passes` will load the page 4 times), so be judicious about adding multiple entries here to avoid extending run times.
+
+Along with the basic settings on how long to wait for the page to load and whether to record a trace file you'll find a list of **gatherers** to use. Gatherers can read information from the page to generate artifacts which are later used by audits to provide you with a Lighthouse report. For more information on implementing a custom gatherer and the role they play in building a lighthouse report, refer to the [recipes](https://github.com/GoogleChrome/lighthouse/blob/master/docs/recipes/).
```

Comment:
link to https://github.com/GoogleChrome/lighthouse/tree/master/docs/recipes/custom-audit